ChangeLog.md

```diff
@@ -4,29 +4,244 @@ Notable changes to the FreeOrion project will be documented in this file.
 
+- Extensive AI improvements
+
+
+### Detailed Changelog
+
+
+#### Graphics / Interface
+
+- Galaxy Map
+    - Improved handling of overlapping fleets and fleet buttons.
+    - Added right-click popup command to dismiss sensor ghosts.
+    - Added option to ignore hostile ships while auto-exploring.
+    - Added ship design breakdown info to fleets indicator at top of screen.
+    - Made pressing escape close open windows in last-opened first-closed order.
+    - Added system shield, defense, troops, and supply summary indicators to top of system sidpanel.
+    - Let fleets window track moving fleets together.
+    - Added an indicator that a fleet is blockaded.
+
+- Research
+    - Items can be deleted from the research queue with ctrl + left click.
+    - Added right click popup commands to research list view rows.
+    - Addded or adjusted tech list view colour highlighting, rounded borders, icon sizes.
+    - Added research list tooltips.
+    - List view scrollbar position should retain its position better.
+   
+- Production
+    - Fixed erroneous early projected completion times for production queue items with multiple repetitions.
+    - Added commands to split or duplicate production items on the queue.
+    - If the selected system is a production item's rally location, this is indicated on the item in the queue.
+
+- Design screen
+    - Ship designs can be reordered, which will affect their ordering on the production screen.
+    - Removed dialog for saving ship designs; a right click popup command is used instead.
+    - Added right-click popup commands to obsolete parts, hulls, or ship designs, or to delete designs.
+    - Added buttons to toggle part / hull obsolete filters.
+    - Indicated part / design availability and obsolescence with desaturated borders in lists.
+    - Added prompt to toggle availability filters when the hulls list is empty.
+    - Added prompt to saved designs to suggest adding some.
+    - Fixed bug where replacing a design twice started added designs.
+    - Allowed monster designs to be viewed, edited, and saved.
+    - When dragging a part over a slot, a part already in the slot should be hidden unless the part in the slot is being dragged.
+    - Ctrl-Dragged parts will remove or replace all parts of the same type.
+    - Removed restriction on setting blank design descriptions.
+
+- Multiplayer
+    - Multiplayer lobby remembers previously-used settings when returning to the lobby after a game.
+    - Added option to limit manipulation of settings in multiplayer lobby to the host.
+    - Added server player name / passwords list, which can require players to password authenticate to join a server.
+        - Players may be assigned roles, which control what they can do in the lobby.
+    - Added optional timestamps to chat in multiplayer lobby and in-game chat.
+    - Ready button in multiplayer lobby will lock if not enough players, and unready if settings were changed by another player.
```

Comment:
As o0leg noted, I think this was in 0.4.7, from PR #1187

Reply:
Yep, seems to be the case. Removed that part.